src/util.rs

```diff
@@ -32,3 +32,38 @@ impl Maximums {
         }
     }
 }
+
+#[macro_export]
+macro_rules! pg_enum {
```

Comment:
this seems like it would be a better fit for the `crates_io::sql` module instead of the generic `util`